### ISSUE TITLE
Build Problem with socket-stream-client@=0.2.1 / Uncaught Error: Cannot find module '/node_modules/meteor/socket-stream-client/sockjs-0.3.4.js'

### ISSUE THREAD

**Issue by DanielDornhardt** [mrauhu/stream-server-uws#1]
Hi,

I tried the whole combination you described here https://forums.meteor.com/t/try-websockets-support-in-the-meteor-pure-speed-no-memory-leaks/45678/2 and it looked really great during development! It gave a great boost to the apps' overall communication stack. Awesome! :D

Unfortunately while live-testing the app with my client, on some of his devices the app didn't function correctly.

The thing is, he's using a mix of iPads from different generations. While the latest version of Safari seems to be compatible socket-stream-client@=0.2.1 pinned, some older tablets won't work (and he can't update them all because of another compatibility with another app... yeah. :) ).

This seems to be the issue described here: https://github.com/meteor/meteor/issues/9928#issuecomment-396257742 , where the fix is in upgrading to the latest / a later `socket-stream-client`.

But that would break the whole construction, or would it?

Basically the issue is a simple import problem with a dependency wrong or missing inside the `socket-stream-client@=0.2.1` - package .

I think only some monkeypatching is needed for the client package, see https://github.com/meteor/meteor/issues/9928#issuecomment-396257742 - but I tried that and that didn't work for me I have to say unfortunately.

I didn't dive much deeper into this issue, but if it's 

a) something you could fix or
b) Maybe you can tell me - would the performance boost still happen without the `stream-client@=0.2.1` pinned? I guess that's required on the server side, right?

I'm gonna try again another time but right now I have to move on to something else unfortunately, I thought I'd drop in and send a report at least.

Thanks & best wishes

Daniel

**Comment by mrauhu** [comment]
@DanielDornhardt Greetings, Daniel.

Thanks for feedback.

I'm added support for latest Meteor versions, and fixed bug that you mention.

Also I created document with step by step instructions:

_Guide for adding support of 🚀 µWebSockets in the Meteor_
https://github.com/mrauhu/meteor-uwebsockets

Best wishes,
Sergey.

**Comment by DanielDornhardt** [comment]
Thank you so much! It's gonna be a great boost for many Meteor Apps for sure.
